src/gwt/src/org/rstudio/core/client/VirtualConsole.java

```diff
@@ -695,14 +732,42 @@ public void ensureStartingOnNewLine()
             submit("\n");
       }
    }
+   private class Hyperlink
+   {
+      public Hyperlink(String url, String params) {
+         url_ = url;
+         params_ = params;
+      }
+
+      public String getTitle() {
```

Comment:
Should this adapt to the type of link ? "Open help page for ...", "Open the viewer ..." ?